README.md

```diff
@@ -286,84 +296,109 @@ The available capture settings are organized into a hierarchy of configuration n
+| `capture_2d/frame_0/iris`          | int    | [Zivid::Settings2D::Iris](https://www.zivid.com/software/releases/1.7.0+a115eaa4-4/doc/cpp/classZivid_1_1Settings2D_1_1Iris.html)
+
+## Samples
+
+In the `zivid_samples` package we have added samples in C++ and Python that demonstrate how to use
+the Zivid ROS driver. These samples can be used as a starting point for your project.
 
 ### Sample Capture
 
-This sample performs single-captures repeatedly. This sample shows how to configure the capture
-settings using [dynamic_reconfigure](https://wiki.ros.org/dynamic_reconfigure), how to subscribe to
-the `points` topic, and how to invoke the `capture` service.
+This sample performs single 3D captures repeatedly. This sample shows how to [configure](#configuration)
+the capture settings, how to subscribe to the `points` topic, and how to invoke the `capture` service.
 
-**C++**  [(Source code)](./zivid_samples/src/sample_capture.cpp)
+Source code: [C++](./zivid_samples/src/sample_capture.cpp), [Python](./zivid_samples/scripts/sample_capture.py)
 
 Using roslaunch (also launches `roscore`, `zivid_camera`, `rviz` and `rqt_reconfigure`):
 ```bash
-roslaunch zivid_samples sample_capture_cpp.launch
+roslaunch zivid_samples sample.launch type:=sample_capture_cpp
+roslaunch zivid_samples sample.launch type:=sample_capture.py
 ```
 Using rosrun (when `roscore` and `zivid_camera` are running):
 ```bash
 rosrun zivid_samples sample_capture_cpp
+rosrun zivid_samples sample_capture.py
 ```
 
-**Python** [(Source code)](./zivid_samples/scripts/sample_capture.py)
+### Sample Capture 2D
+
+This sample performs 2D captures repeatedly. This sample shows how to [configure](#configuration)
+the 2D capture settings, how to subscribe to the `color/image_color` topic, and how to invoke
+the `capture_2d` service.
+
+Source code: [C++](./zivid_samples/src/sample_capture_2d.cpp), [Python](./zivid_samples/scripts/sample_capture_2d.py)
 
 Using roslaunch (also launches `roscore`, `zivid_camera`, `rviz` and `rqt_reconfigure`):
 ```bash
-roslaunch zivid_samples sample_capture_py.launch
+roslaunch zivid_samples sample.launch type:=sample_capture_2d_cpp
+roslaunch zivid_samples sample.launch type:=sample_capture_2d.py
 ```
 Using rosrun (when `roscore` and `zivid_camera` are running):
 ```bash
-rosrun zivid_samples sample_capture.py
+rosrun zivid_samples sample_capture_2d_cpp
+rosrun zivid_samples sample_capture_2d.py
 ```
 
 ## Sample .launch files
 
 ### zivid_camera_with_settings.launch
 
 [zivid_camera_with_settings.launch](./zivid_samples/launch/zivid_camera_with_settings.launch) starts
-the `zivid_camera` node with capture settings set to 3-frame HDR. Modify the settings in this .launch
+the `zivid_camera` node with 3D capture settings set to 3-frame HDR. Modify the settings in this .launch
 file to match your scene.
 
 ```bash
```

Comment:
This is outside the scope of this PR, but: How does one then actually visualize this 3-frame HDR?

I don't know how to combine it with the visualization-instructions below.

Reply:
Should work to 
roslaunch zivid_samples zivid_camera_with_settings.launch
ROS_NAMESPACE=zivid_camera roslaunch zivid_camera visualize.launch
then trigger capture
rosservice call /zivid_camera/capture